manifests/install.pp

```diff
@@ -210,7 +212,7 @@
   # The install should run only once for a given version. This creates
   # the Shibboleth install base (by default /opt/shibboleth-dip).
   exec { 'shibboleth idp install':
-    command     => "${shibboleth_idp::shib_src_dir}/shibboleth-identity-provider-${shibboleth_idp::shib_idp_version}/bin/install.sh  -Didp.property.file=${shibboleth_idp::shib_src_dir}/idp.install.properties",
+    command     => "${shibboleth_idp::shib_src_dir}/shibboleth-identity-provider-${shibboleth_idp::shib_idp_version}/bin/install.sh  -Didp.property.file=${shibboleth_idp::shib_src_dir}/idp.install.properties", # lint:ignore:140chars
```

Comment:
many ignore this globally in Rakefile

Reply:
https://github.com/puppetlabs/pdk-templates#rakefile
`default_disabled_lint_checks`